cmd/lima

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 set -eu
 : "${LIMA_INSTANCE:=default}"
+: "${LIMA_SHELL:=}"
```

Comment:
I'm not sure we want to have this env var.
A user should just chsh.

Reply:
> A user should just chsh.

Sure, but you have to reboot before it takes effect (because we enable "linger"), and it is not obvious to the user, so it looks like it doesn't work at first. Using the env variable makes it easy to switch between shells.

Reply:
Wondering rather we should have a generic way to support `limactl shell` options in the `lima` script. Perhaps like `LIMA_SHELL_ARGS=--workdir=/ --shell=/bin/zsh`. But on the second thought maybe users should just set `alias lima=limactl shell ....`.


Could you split the `lima` script into another PR for further discussion?

Reply:
> Could you split the lima script part into another PR for further discussion?

Moved to #780